apps/demo/src/app/drivers/in-memory/backend/backend.service.ts

```diff
@@ -33,8 +33,8 @@ export class DemoInMemoryBackendService implements InMemoryDbService {
   }
 
   post(reqInfo: any) {
-    const collectionName = reqInfo.collectionName;
-    if (collectionName === 'cart') {
+		const collectionName = reqInfo.collectionName;
+    if (collectionName.includes('cart')) {
```

Comment:
The list of collection names that cart accepts is provided as a static field so you can do this check like `DaffInMemoryBackendCartRootService.COLLECTION_NAMES.indexOf(collectionName) > -1`. Its more verbose but less brittle.

Reply:
changed